content/telegraf/v1.20/administration/commands.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Telegraf commands and flags
+description: The `telegraf` command starts and runs all the processes necessary for Telegraf to function.
+menu:
+  telegraf_1_20:
+    name: Commands
+    weight: 20
+    parent: Administration
+---
+
+The `telegraf` command starts and runs all the processes necessary for Telegraf to function.
+
+## Usage
+
+```
+telegraf [commands]
+telegraf [flags]
+```
+
+## Commands
+
+
+
+| Command   | Description                                    |
+| :-------- | :--------------------------------------------- |
+| `config`  | Print out full sample configuration to stdout. |
+| `version` | Print version to stdout.                       |
+
+## Flags
+
+| Flag                             | Description                                                                                                                       |
+| :------------------------------- | :-------------------------------------------------------------------------------------------------------------------------------- |
+| `--aggregator-filter <filter>`   | Filter aggregators to enable. Separator is `:`.                                                                                   |
+| `--config <file>`                | Configuration file to load.                                                                                                       |
+| `--config-directory <directory>` | Directory containing additional `*.conf` files                                                                                    |
+| `--watch-config`                 | Restart Telegraf on local config changes. Use either fs notifications (`inotify`) or polling (`poll`). Disabled by default        |
```

Comment:
does it make sense to spell out config for consistency, and use `config`only when referring to commands or file extension?